org.eclipse.xtext/src/org/eclipse/xtext/generator/trace/node/TracedAccessorsProcessor.java

```diff
@@ -63,6 +63,7 @@ public void doTransform(MutableClassDeclaration annotatedClass, @Extension Trans
 					if (TracedAccessorsProcessor.TYPES_WITH_GOOD_TO_STRING
 							.contains(rt.getType().getSimpleName().toLowerCase())) {
 						annotatedClass.addMethod(tracerName(getter), it -> {
+							context.setPrimarySourceElement(it, context.getPrimarySourceElement(annotatedClass));
```

Comment:
Is this intentionally part of this merge request?

Reply:
No